Makefile

```diff
@@ -76,6 +76,11 @@ shell:
 	-v ${PWD}:/go/src/github.com/kris-nova/kubicorn \
 	--rm ${SHELL_IMAGE} /bin/bash
 
+lint:
+	which golint > /dev/null || go get -u github.com/golang/lint/golint...
```

Comment:
I took this from the example on line 23, it also has the three dots. Can remove them because it looks like it doesn't really affect the functionality.